packages/docs/versioned_docs/version-2.0.0-alpha-3/protocol/05-user-state-transition.md

```diff
@@ -4,43 +4,39 @@ description: Definition of user state transition in UniRep protocol.
 
 # User State Transition
 
-### Why users have to perform user state transition?
+### Why do users have to perform user state transition?
 
-User state transition is used to&#x20;
+User state transition is used to:
 
-* Make sure users process their attestations correctly including the bad reputation.
+* Ensure users process their attestations correctly, including negative reputation.
 * Generate a new state tree leaf in a new epoch to prove their latest reputation.
 
-After user performs user state transition, he can&#x20;
+After a user performs user state transition, they can:
 
-1. Prove the latest reputation status.
+1. Prove their latest reputation status.
 2. Generate new epoch key proofs to receive attestations in the latest epoch.
 
 ### Workflow of a user state transition
 
-#### 1. User computes epoch key of the latest transition (or sign up) epoch
-
+#### 1. Compute epoch key of the latest transition (or sign up) epoch
 
 #### 2. Check the state tree leaf
-
-The pre-image of the state tree leaf is checked to determine the users starting reputation.
+* The pre-image of the state tree leaf is checked to determine the user's starting reputation.
 
 #### 3. Iterate over epoch keys
 
-The pre-image of the epoch key leaves controlled by the user are verified. During this the `posRep` and `negRep` is summed and the `graffiti` is combined by taking the graffiti with the latest `timestamp`.
-
-The summed rep values are added to the values proved from the state tree leaf.
-
+* The pre-image of the epoch key leaves controlled by the user are verified. `posRep` and `negRep` are summed and `graffiti` is combined by taking the graffiti with the latest `timestamp`.
```

Comment:
does this mean that multiple graffiti values are combined together, or that only the latest is used and others discarded?

Reply:
Only the latest is used and the others are discarded.